components/Accordion/mock.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Square = styled.div`
+  background: black;
+  height: 32px;
+  width: 32px;
+  display: inline-block;
+  margin-right: 8px;
+`;
+
+export default {
+  items: [
+    {
+      title: 'Institucional',
+      content: (
+        <div>
+          Blog <br />
```

Comment:
Adding some `List` examples would be nice.

Reply:
Done